App.js

```diff
@@ -1,46 +1,73 @@
 import React from "react";
-import AppContainer from "./src/navigation";
-import AppLoading from "expo-app-loading";
-import * as Font from "expo-font";
-import { Asset } from "expo-asset";
-
-export default class App extends React.Component {
-  //Custom fonts
-  state = {
-    isReady: false,
-  };
-
-  async _cacheResourcesAsync() {
-    return new Promise(async (resolve) => {
-      try {
-        await Font.loadAsync({
-          "open-sans": require("./src/assets/fonts/OpenSans-Regular.ttf"),
-          "open-sans-bold": require("./src/assets/fonts/OpenSans-Bold.ttf"),
-          alfaSlabOne: require("./src/assets/fonts/AlfaSlabOne-Regular.ttf"),
-        });
-      } catch (error) {
-        console.log(error);
-      }
-
-      resolve();
-      const images = [require("./src/assets/splash.png")];
-
-      const cacheImages = images.map((image) => {
-        return Asset.fromModule(image).downloadAsync();
-      });
-    });
-  }
+import { StatusBar as ExpoStatusBar } from "expo-status-bar";
+import firebase from "firebase/app";
+// import AppLoading from "expo-app-loading";
+// import * as Font from "expo-font";
+// import { Asset } from "expo-asset";
```

Comment:
I suggest we go about removing these imports rather than commenting them out.

Reply:
Will do!